gateway/crates/federated-server/src/server/gateway.rs

```diff
@@ -146,30 +148,36 @@ pub struct GatewayRuntime {
     hooks: HooksWasi,
     rate_limiter: runtime::rate_limiting::RateLimiter,
     entity_cache: Box<dyn EntityCache>,
+    operation_cache_factory: InMemoryOperationCacheFactory,
 }
 
 impl engine_v2::Runtime for GatewayRuntime {
     type Hooks = HooksWasi;
     type Fetcher = NativeFetcher;
-    type OperationCacheFactory = ();
+    type OperationCacheFactory = InMemoryOperationCacheFactory;
 
     fn fetcher(&self) -> &Self::Fetcher {
         &self.fetcher
     }
+
     fn trusted_documents(&self) -> &runtime::trusted_documents_client::Client {
         &self.trusted_documents
     }
+
     fn kv(&self) -> &runtime::kv::KvStore {
         &self.kv
     }
+
     fn meter(&self) -> &grafbase_telemetry::otel::opentelemetry::metrics::Meter {
         &self.meter
     }
+
     fn hooks(&self) -> &HooksWasi {
         &self.hooks
     }
+
     fn operation_cache_factory(&self) -> &Self::OperationCacheFactory {
-        &()
+        &self.operation_cache_factory
```

Comment:
Did you manage to clarify why that was a no-op?

Reply:
No. There's a PR

https://github.com/grafbase/grafbase/pull/1854

This PR is about adding back the op cache. But it did not do it eventually...